src/main/index.js

```diff
@@ -31,19 +34,30 @@ if (process.platform === 'win32') {
   const squirrelWin32 = require('./squirrel-win32')
   shouldQuit = squirrelWin32.handleEvent(argv[0])
   argv = argv.filter((arg) => !arg.includes('--squirrel'))
+
+  if (shouldQuit) {
+    app.quit()
+  } else {
+    app.on('second-instance', (event, commandLine) => onAppOpen(commandLine))
+  }
```

Comment:
Why is this logic repeated below? It seems like this will cause the `'second-instance'` event to be registered twice. Is this intentional?